src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+
+.calendar {
+  position: relative;
+  top: 50%;
+  left: 50%;
```

Comment:
yes it is, but @viktoriia-kravchenko is right, these styles are external position of calendar, so use mix here

Reply:
I got it, thx